### DISCUSSION TITLE
Signing Custom Build - Verifying / Notarizing requiring Sandbox?

### DISCUSSION THREAD

**Discussion by marcusheBG** [IBM/mac-ibm-notifications#144] [Q&A]
Hello! Thank you so much in advance.

We are successfully signing a custom version of IBM Notifier with our Developer ID. However in order to verify the sign - Apple is saying that the app is required to be sandboxed. We have added the App Sandbox payload and for testing only - checked permissions for all items. However, after building with the Sandbox payloads added - the compiled custom Notifier app will run / open, but not accept any terminal commands and perform any functions.

However, the app runs perfectly fine with a normal signature as well.

Any ideas or guidance? Thank you!

- Marcus

<img width="1659" alt="Screenshot 2022-12-05 at 11 52 47 AM" src="https://user-images.githubusercontent.com/90919338/205698849-6b8c4291-17f5-4e4d-b11e-5ad220250771.png">

<img width="1789" alt="Screenshot 2022-12-05 at 11 53 12 AM" src="https://user-images.githubusercontent.com/90919338/205698379-9ba3b3c3-f5dc-41df-997b-e3cb81d8f8a3.png">

<img width="1161" alt="screenshot_2022-12-02_at_11 07 44_am-1" src="https://user-images.githubusercontent.com/90919338/205698181-bb628968-9b2e-427f-9d87-8ba28c94e9c1.png">

**Comment by SMartorelli** [comment]
Hi @marcusheBG I see that you also enabled all the exception in the Hardened Runtime capability tab, it's not needed. IBM Notifier don't need to have sandbox capability so I suggest you to restart from the scratch from the repo code. It's hard to debug signature/validation issues without the code.
PS: you're trying to sign it with an Enterprise Developer Account right?
